Imagine/Filter/PostProcessor/OptiPngPostProcessor.php

```diff
@@ -44,7 +64,16 @@ public function process(BinaryInterface $binary)
         }
 
         $pb = new ProcessBuilder(array($this->optipngBin));
-        $pb->add('--o7');
+
+        if ($this->level !== null) {
+            $pb->add(sprintf('--o%d', $this->level));
+        }
+
+        if ($this->stripAll) {
+            $pb->add('--strip=all');
+        }
+
+        $pb->add($input = tempnam(sys_get_temp_dir(), 'imagine_optipng'));
```

Comment:
Sorry for my late comment, but I have noticed that there is merge conflict. $input gets added twice.